addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -31,12 +31,13 @@
                         widget="timer_toggle_button"
                         readonly='True'
                         attrs="{
-                            'invisible': [('encoding_uom_name', '!=', 'Hours')]
+                            'invisible': &quot;['|', ('encoding_uom_name', '!=', 'Hours'), ('user_id', '!=', uid)]&quot;
```

Comment:
Small question: And if a user can see all the timesheets. Do we cant the same restriction ? Maybe if we want to restrict according to the group, we should introduce a simple computed field (non stored) to display the button or not.

To check with (aju).

Reply:
In fact, the basic idea is, I must to only display this timer button in the menu ''My Timesheet", this condition was for this. But I have some problems with this condition in the kanban view, than I use a "magic context" as @LucasLefevre remarked it in a next commit. Perhaps, there is another solution, I will search this.